docker-compose-dev.yml

```diff
@@ -16,13 +16,12 @@ services:
       ENABLE_RELOAD: "1"
       SKIP_DEPLOY: "${SKIP_DEPLOY}"
       DATAVERSE_JSF_REFRESH_PERIOD: "1"
-      DATAVERSE_FEATURE_API_BEARER_AUTH: "1"
```

Comment:
We enabled bearer auth in bb1f64e83a02ff757b835c30f07caac30b3204d1 and in that commit explained how to test it. Why are we removing it?